packages/sanity-plugin-nacelle-input/README.md

```diff
@@ -0,0 +1,147 @@
+# Sanity Custom Input Plugin: Nacelle PIM Linker
+
+[Nacelle](https://docs.getnacelle.com/intro.html#what-is-nacelle) indexes data from your [PIM](https://en.wikipedia.org/wiki/Product_information_management) (e.g. Shopify, Magento) and CMS to power headless eCommerce projects. This plugin provides a [custom input component](https://www.sanity.io/docs/custom-input-widgets) for Sanity Studio that helps you reference product & collection data stored in Nacelle indices.
+
+<details>
+  <summary>Expand to see the custom input component in action!</summary>
+  <img src="https://user-images.githubusercontent.com/5732000/105260780-65532a00-5b5c-11eb-9cc5-c5f8bddb89b4.gif" alt="The Nacelle PIM Linker component is used in Sanity Studio to select products stored in Nacelle's indices">
+</details>
+
+## Installation & Setup
+
+### Install Peer Dependencies
+
+`npm i @sanity/ui styled-components`
+
+### Install the plugin
+
+`sanity install @nacelle/sanity-plugin-nacelle-input`
+
+### Credentials
+
+You'll need to provide the ID and Token associated with your Nacelle space. These credentials can be found in the [Nacelle Dashboard](https://dashboard.getnacelle.com/).
+
+#### Adding credentials to a single space:
+
+##### Config file for single space - `./config/@nacelle/sanity-plugin-nacelle-input.json`
+
+```json
+{
+  "nacelleSpaceEndpoint": "your-nacelle-storefront-api-endpoint",
```

Comment:
To match what's expected in the code (for example, in `src/hooks/useHailFrequency.js`):

```suggestion
  "nacelleEndpoint": "your-nacelle-storefront-api-endpoint",
```